modules/plan.nix

```
@@ -130,8 +130,8 @@ let
     };
     # Wrapper for test executable run in checkPhase
     testWrapper = mkOption {
-      type = str;
-      default = (def.testWrapper or "");
+      type = nullOr str;
+      default = (def.testWrapper or null);
```

```suggestion
      default = def.testWrapper or null;
      description = "A command to run for executing tests in checkPhase, which takes the original test command as its arguments.";
      example = "echo";
```

Let's start documenting options as we touch them.